buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -40,12 +41,13 @@ object Versions {
     object Mod {
         const val mixin = "0.15.3+mixin.0.8.7"
         
-        const val minecraft = "1.21.1"
-        const val yarn = "$minecraft+build.3"
-        const val fabricLoader = "0.16.5"
+        const val minecraft = "1.21.2-pre3"
```

Comment:
Could probably be bumped to `1.21.2` by now

Reply:
I am about to push a bump to 1.21.3